pythoncalculator/multiply.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x,y);
```

Comment:
I think you mistaken ; for :

```suggestion
def multiply(x,y):
```